### PR TITLE
Add section for configuring Okta as an auth backend

### PR DESCRIPTION
**Pull request by phoebesimon**
*What*
Adds a section under `Reference` on configuring Okta auth

*How*
teams.md: add blurbs on Okta and SAML auth backend configuration options
setup-okta-team.md: detailed instructions on how to configure a team with Okta

*Why*
It's kinda complicated for someone to figure out on their own

I wanted to add in pictures too, but wasn't sure exactly how to go about that. Also, not entirely sure if this is the place to make these changes or if there's somewhere else I need to change. Guidance would be awesome

### CODE REVIEW

**Comment by ggreer** [comment]
For screenshot examples, see `sft-osx.md` and `enrolling-a-client.md`. Basically:

- Resize the window to something sane, if it's a browser, hide your extensions.
- Screenshot with ⌘ + ⇧ + 4, then hit spacebar, then click on the window you want to capture.
- Copy the screenshot into `docs/static`.
- Set a max-height on the image that is 50% of the actual height in pixels. This is because retina screenshots are at 2x resolution. Don't set a height or width, as that will mess up the aspect ratio on narrow screens.

If you have any terminal output that you'd like to display, search for `{{% terminal %}}` to find examples.

File: setup-okta-team.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    parent: 'reference'
+    weight: 2
+---
+
+1. After choosing your teamname and billing plan, select the "Okta" option for the auth backend and click "Next". You should see a page that tells you what values to enter for "Base URL" and "Audience URI"
+
+<img src="/docs/static/okta-config-page.png" style="max-height: 621px;" />
+
+2. In a separate tab, log in to your Okta instance
+3. Click the "Admin" button in the upper right corner
```
Comment (ggreer) [inline setup-okta-team.md]: Hugo seems to be changing these numbers to `1.`. Unless you know how to stop hugo from being dumb, I guess just kill the numbers and leave the sentences between images.

Comment (jbeverly) [inline setup-okta-team.md]: escape the '.' ala 1\.

Comment (phoebesimon) [inline setup-okta-team.md]: @jbeverly like so? 	e979d89

File: setup-okta-team.md [hunk 2]
```diff
@@ -6,50 +6,50 @@ menu:
...
 
-1. After choosing your teamname and billing plan, select the "Okta" option for the auth backend and click "Next". You should see a page that tells you what values to enter for "Base URL" and "Audience URI"
+1\. After choosing your teamname and billing plan, select the "Okta" option for the auth backend and click "Next". You should see a page that tells you what values to enter for "Base URL" and "Audience URI"
 
 <img src="/docs/static/okta-config-page.png" style="max-height: 621px;" />
 
-2. In a separate tab, log in to your Okta instance
-3. Click the "Admin" button in the upper right corner
+2\. In a separate tab, log in to your Okta instance
+3\. Click the "Admin" button in the upper right corner
```
Comment (ggreer) [inline setup-okta-team.md]: These end up on the same line.

screen shot 2016-05-06 at 14 11 30

I think most markdown parsers treat two spaces at the end of a line as "no, really, I mean newline here".
Or you can add another newline between them. That will start a new `<p>`.

Comment (phoebesimon) [inline setup-okta-team.md]: @ggreer fixed in 1dc0a3a

**Comment by ggreer** [comment]
+1

**Pull request merged**